.github/workflows/nightly.yaml

```diff
@@ -65,11 +65,9 @@ jobs:
 
     - name: Sign released images
       env:
-        # This enables keyless mode
-        # (https://github.com/sigstore/cosign/blob/main/KEYLESS.md) which signs
-        # images using an ephemeral key tied to the GitHub Actions identity via
-        # OIDC.
-        COSIGN_EXPERIMENTAL: "true"
+        # This enables keyless mode (https://docs.sigstore.dev/cosign/keyless/) which signs
+        # images using an ephemeral key tied to the GitHub Actions identity via OIDC.
+        COSIGN_EXPERIMENTAL: "1"
```

Comment:
from https://blog.sigstore.dev/cosign-2-0-released/ I don't think this env var is required anymore, wdyt?

Reply:
That's nice.